src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -100,41 +100,60 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
     });
   }
 
-  public getToolTipContent(item: any, queryContent: any) {
-    let selectionDisabled = false;
-
-    if (!this.props.tokenPresent && item.method !== 'GET') {
-      selectionDisabled = true;
-    }
-
-    return selectionDisabled ? <FormattedMessage id={'Sign In to try this sample'} /> : queryContent;
-
-  }
-
   public renderItemColumn = (item: any, index: number | undefined, column: IColumn | undefined) => {
     const classes = classNames(this.props);
-    const hostId: string = getId('tooltipHost');
 
     if (column) {
       const queryContent = item[column.fieldName as keyof any] as string;
 
-      const toolTipContent = this.getToolTipContent(item, queryContent);
-
       switch (column.key) {
+        case 'authRequiredIcon':
+          if (item.method !== 'GET') {
+            return <TooltipHost
+            tooltipProps={{
+              onRenderContent: () => <div style={{ paddingBottom: 3 }}>
+                <FormattedMessage id={'Sign In to try this sample'} /></div>
```

Comment:
>sample [](start = 59, length = 6)

this is before your change but my only problem with this word is that if you do sign in and run this command, it's not really a sample.  You are modifying your actual data.

Reply:
Should we change this to request instead?

Reply:
maybe but i'll leave to PM to decide if it should be changed at all.  It's pre-existing so I'm fine as is in this PR.